### ISSUE TITLE
[edn/sw] Return the whole sts code in dif_edn_get_status

### ISSUE THREAD

**Issue by h-filali** [lowRISC/opentitan#22465]
### Description

Currently dif_edn_get_status sets the *set argument to 1 in case of an error and to 0 in case of success.
It would be better if the set bit could be replaced by the whole sts response signal, i.e. the whole status code (which is 0 in case of success and a positive value representing the status in the error case).

**Comment by vogelpi** [comment]
Thanks for creating this issue @h-filali . This is not critical for the current hardware milestones. We need to check if this is relevant for ROM. I am now moving to that milestone.

**Comment by moidx** [comment]
Hi @vogelpi, currently the ROM only uses boot mode entropy. Should we move this to the ROM_EXT? That is the first time we do a full entropy complex configuration in continuous mode. If you agree, please move to the ROM_EXT milestone. Thanks!

**Comment by vogelpi** [comment]
Yes, that sounds fair to me @moidx . In ROM, the whole entropy complex runs in hardware defined mode. Software cannot operate it in the wrong way, so there is no strict need to infer status flags.
